content/en/cloud_cost_management/multisource_querying/_index.md

```diff
@@ -23,10 +23,7 @@ Use Multisource Querying to build cost views, understand the total cost of servi
 
 ## Setup
 
-To use Multisource Querying, ensure you meet the following requirements:
-
-1. You have configured [Cloud Cost Management][10] and are actively ingesting costs in Datadog.
-1. Your cost data is in a single currency. Multiple currencies are not supported.
+To use Multisource Querying, ensure you have configured [Cloud Cost Management][10] and are actively ingesting costs in Datadog. Multiple currencies are now supported, with your costs automatically being converted to and displayed in USD.
```

Comment:
```suggestion
To use Multisource Querying, ensure you have configured [Cloud Cost Management][10] and are actively ingesting costs in Datadog. Multiple currencies are supported, with your costs automatically being converted to and displayed in USD.
```

No reason to write `now`. I dont think docs should have a sense of past features in them. That is more something for a change log, which we do not have